### ISSUE TITLE
Tables missing headers 

### ISSUE THREAD

**Issue by ghost** [cis-bathnes/pattern-lib#11]
![image](https://user-images.githubusercontent.com/609519/34771616-0c3d2f96-f5fd-11e7-93c8-4e872fb8a69c.png)
Row headers (with class "bold") should be marked up using th elements.
See: https://www.w3.org/WAI/tutorials/tables/one-header/ (example 2)

Reference: Templates > Check Your Answers

**Comment by BenDarby** [comment]
Used `<th>` for table headers
